site-src/v1alpha2/api-types/referencegrant.md

```diff
@@ -1,35 +1,39 @@
-# ReferencePolicy
+# ReferenceGrant
 
-A ReferencePolicy can be used to enable cross namespace references within
+!!! note
+    This resource was originally named "ReferencePolicy". It was renamed
+    to "ReferenceGrant" to avoid any confusion with policy attachment.
```

Comment:
Should a note like this be added to GEP-709 too?

Reply:
Updated, thanks!